### DISCUSSION TITLE
How to remove reference from Kindle Highlights

### DISCUSSION THREAD

**Discussion by ravipawar89** [hadynz/obsidian-kindle-plugin#206] [Q&A] [answered]
<img width="101" alt="image" src="https://user-images.githubusercontent.com/11712495/193437254-b8b29309-66fe-4913-90ba-fe4426458ba0.png">

I want to remove references which are coming for each highlighted text, tried with setting variable change and still these are coming. Please help with the solution on this as I tried all

**Comment by hadynz** [comment, marked as answer]
This is by design. These references are used by the plugin to "understand" which clipping is on this line so it can re-sync any new clippings for a book that you might be reading.

Currently there is no way to disable these automatically generated references.

**Reply by ravipawar89** [reply]
ohk... then this makes sense... actually i was trying to use all my highlights on logseq instead of obsidian and in logseq your plugin is not there.
anyways thanks for your prompt response

**Reply by hadynz** [reply]
Oh interesting. I've never used logseq before. I'll keep an eye on other requests and see if it makes sense to port this plugin to logseq.
